example.py

```diff
@@ -1,5 +1,8 @@
 def add(a,b);
-return a + b
+  return a + b
 
-def subtract(a,b)
-return a - b
+def subtract(a,b):
+  return a - b
+
+def multithingy(a,b):
+  return a*a
```

Comment:
return a * a would be better 📦

Reply:
thanks, do you think I should pass just one input to the function?

Reply:
HI